src/components/HeaderInfo/HeaderInfo.js

```diff
@@ -215,6 +215,10 @@ export const HeaderInfo = ({
   const [selectedReportingPeriods, setSelectedReportingPeriods] = useState(
     currentTab?.reportingPeriods ?? []
   );
+  /* defaultReportingPeriodChanged is a flage; user selecte a reporing date or not;
+    it's changed when user manually change location or viewTemplate or when import data
+  */
```

Comment:
Some of the spelling is off here:
```suggestion
  /* defaultReportingPeriodChanged is a flag for whether a user has selected a reporing date or not;
    it's changed when user manually changes location or viewTemplate or imports data
  */
```